src/formats/notion.ts

```diff
@@ -102,9 +102,13 @@ export class NotionImporter extends FormatImporter {
 					ctx.status(`Importing note ${fileInfo.title}`);
 
 					const markdownBody = await readToMarkdown(info, file);
+					const writeOptions: DataWriteOptions = {
+						ctime: fileInfo.ctime ? fileInfo.ctime.getTime() : Date.now(),
+						mtime: fileInfo.ctime ? fileInfo.ctime.getTime() : Date.now(),
```

Comment:
How come `mtime` is also set to `ctime`?

Reply:
If I import Notion zip file with the current version, both ctime and mtime are the current time. So it will not be able to sort in the Dataview. Just in case, even if fs uses ctime as the current time, I can sort with mtime.

Reply:
I'm hesitant to set the last modified time to be the file's original creation time. I am assuming this information is not available anywhere in the export?

Reply:
Typically, most users don't use the last edited time property, but I've changed the code to use the last edited time if it exists.
<tr class="property-row property-row-created_time"> <tr class="property-row property-row-created_time
<tr class="property-row property-row-last_edited_time"> <tr class="property-row property-row-last_edited_time